File: 0009-palindrome-number/solution.java
import java.io.*;
import java.util.*;
class Solution {
    public boolean isPalindrome(int x) {
        String z=String.valueOf(x);
        String temp="";
        char c;
        for(int i=0;i<z.length();i++)
        {
            c=z.charAt(i);
            temp=c+temp;
        }
      return z.equals(temp);
    }
}

File: 0009-palindrome-number/solution.c
bool isPalindrome(int x) {
    int ans=x;
    long int sum=0;
    while(x>0)
    {
        int r=x%10;
        sum=sum*10+r;
        x=x/10;
    }
    if(ans==sum)
        return true;
    return false;
}
